app/ui/content/index.jsx

```diff
@@ -10,14 +10,20 @@
  specific language governing permissions and limitations under the License.
  */
 
-import { bindHistorySearch, fetchRecentHistory } from './views/history';
-import { fetchRecentBookmarks } from './views/stars';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
 
-if (~document.location.pathname.indexOf('history')) {
-  bindHistorySearch();
-  fetchRecentHistory();
-}
+import App from './views/app';
+import { createContentStore } from './store';
 
-if (~document.location.pathname.indexOf('stars')) {
-  fetchRecentBookmarks();
-}
+const store = createContentStore();
+
+const chrome = (
```

Comment:
'chrome' name seems overloaded here, probably shouldn't show up at all in this app